openedx/features/offline_mode/assets_management.py

```diff
@@ -0,0 +1,162 @@
+"""
+This module contains utility functions for managing assets and files.
+"""
+import shutil
+import logging
+import os
+import requests
+
+from django.conf import settings
+from django.core.files.base import ContentFile
+from django.core.files.storage import default_storage
+
+from xmodule.assetstore.assetmgr import AssetManager
+from xmodule.contentstore.content import StaticContent
+from xmodule.exceptions import NotFoundError
+from xmodule.modulestore.exceptions import ItemNotFoundError
+
+from .constants import MATHJAX_CDN_URL, MATHJAX_STATIC_PATH
+
+
+log = logging.getLogger(__name__)
+
+
+def get_static_file_path(relative_path):
```

Comment:
We probably will need to use utils from other modules to resolve static for sure

Reply:
If you mean to use an existing method for this, then unfortunately I couldn't find one, because it uses `STATIC_ROOT`, not `STATIC_URL`.

And if you mean to use this method from another part of the platform, then there shouldn't be any problems with that, because `assets_management` doesn't have imports from LMS and CMS